gradle.properties

```diff
@@ -1,5 +1,5 @@
 VERSION_CODE=1
```

Comment:
Hm, okay. I've always bumped it when bumping versions but mostly out of ceremony and not necessarily knowing what it does (although I always thought it might have mattered for some maven dependency related reasons).